### PR TITLE
Sprint 72

### PR DESCRIPTION
**Pull request by andriisoldatenko**

### CODE REVIEW

File: bekindred/templates/base_dev_ng2.html [hunk 1]
```diff
@@ -79,4 +79,8 @@
     $('.connect-media').trigger('persice:refreshConnect');
   }
   </script>
+  <script type="text/javascript">
+      window.heap=window.heap||[],heap.load=function(e,t){window.heap.appid=e,window.heap.config=t=t||{};var n=t.forceSSL||"https:"===document.location.protocol,a=document.createElement("script");a.type="text/javascript",a.async=!0,a.src=(n?"https:":"http:")+"//cdn.heapanalytics.com/js/heap-"+e+".js";var o=document.getElementsByTagName("script")[0];o.parentNode.insertBefore(a,o);for(var r=function(e){return function(){heap.push([e].concat(Array.prototype.slice.call(arguments,0)))}},p=["clearEventProperties","identify","setEventProperties","track","unsetEventProperty"],c=0;c<p.length;c++)heap[p[c]]=r(p[c])};
```
Comment (andriisoldatenko) [inline bekindred/templates/base_dev_ng2.html]: @sasha7 Please review

File: bekindred/templates/base_dev_ng2.html [hunk 2]
```diff
@@ -27,6 +27,11 @@
 
   <meta name="description" content="">
   <meta name="author" content="">
+  <script>
+      (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',intercomSettings);}else{var d=document;var i=function(){i.c(arguments)};i.q=[];i.c=function(args){i.q.push(args)};w.Intercom=i;function l(){var s=d.createElement('script');s.type='text/javascript';s.async=true;
```
Comment (andriisoldatenko) [inline bekindred/templates/base_dev_ng2.html]: @sasha7 Please review!

File: bekindred/templates/base_dev_ng2.html [hunk 3]
```diff
@@ -27,6 +27,11 @@
...
+          s.src='https://widget.intercom.io/widget/{app_id}';
```
Comment (andriisoldatenko) [inline bekindred/templates/base_dev_ng2.html]: @sasha7 Also we need to provide correct app_id

File: bekindred/templates/base_dev_ng2.html (hunk 2 above)
Comment (sasha7) [inline bekindred/templates/base_dev_ng2.html]: looks ok!

File: bekindred/templates/base_dev_ng2.html (hunk 3 above)
Comment (sasha7) [inline bekindred/templates/base_dev_ng2.html]: looks ok!

File: bekindred/templates/base_dev_ng2.html (hunk 1 above)
Comment (sasha7) [inline bekindred/templates/base_dev_ng2.html]: looks ok!

**Pull request merged**